core/templates/dev/head/pages/topic_editor/questions/QuestionsTabDirective.js

```diff
@@ -32,6 +32,8 @@ oppia.directive('questionsTab', [
             UndoRedoService, UrlInterpolationService,
             EVENT_STORY_SUMMARIES_INITIALIZED, EVENT_TOPIC_INITIALIZED,
             EVENT_TOPIC_REINITIALIZED) {
+          // TODO(aks681): Continue with the questions editor frontend in this
```

Comment:
Btw, just a note for the future: you want this to be more of a QuestionEditorDirective (and question_tab_directive would be a separate template that includes QuestionEditorDirective). Not all the pages you have a question editor directive will use tabs, I think (e.g. if we do an "add question" modal popup in the creator dashboard).

Reply:
Yeah, that's what I was planning to do. Have the question editor directive be called in this tab. Will make the comment clearer.

Reply:
Done.